src/components/ViewRequest/Details.tsx

```diff
@@ -40,11 +43,28 @@ const ViewRequestDetails = ({
     "Undefined",
   ].includes(request.data?.stage || "Undefined");
 
+  const areAnnounceMethodsAddable = isHRL || isCOSF;
```

Comment:
AC1 states you should also add in that in must be in one of the 2 acceptable stage